modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -662,48 +663,59 @@ bool Visualizer::key_handler(const std::string &camera_name, const int key) {
     case 86: case 118:  // 'V' 'v'
       show_velocity_ = (show_velocity_ + 1) % 2;
       break;
-    case 65362:  // Up_Arrow
-      if (pitch_adj_degree_ + 0.05 <= max_pitch_degree_) {
+    case 1113938: case 65362:  // Up_Arrow with Num Lock (on/off)
```

Comment:
Better to name these key code as const variables ahead and use them, like
```
enum KeyCode {
  KEY_UPPER_V = 86,
  KEY_LOWER_V = 118,
  KEY_UP_ARROW = 1113938,
  ...
}
```
I guess there are some third-party libraries providing these values, which would be even better.

Reply:
Thank you for the review. I modified the code to have enum. I checked 3rd party libraries but I think it is better to keep our own to avoid any dependency or copyright issues.